core/client/hdfs/src/main/java/alluxio/hadoop/LocalCacheFileSystem.java

```diff
@@ -130,8 +131,15 @@ public FSDataInputStream open(Path path, int bufferSize) throws IOException {
         .setGroup(externalFileStatus.getGroup());
     // FilePath is a unique identifier for a file, however it can be a long string
     // hence using md5 hash of the file path as the identifier in the cache.
-    CacheContext context = CacheContext.defaults().setCacheIdentifier(
-        md5().hashString(externalFileStatus.getPath().toString(), UTF_8).toString());
+    String cacheIdentifier;
```

Comment:
Looks good.
Why don't we enable this by default?
I don't see the necessity of introducing another property key.

Reply:
For me, I am not clear that which third-party systems are using this class and if `externalFileStatus.getModificationTime()` has a meaningful value for them.

I got the suggestion from beinan that adding a property key should be better. Maybe @beinan can provide some insights?

Reply:
OK.